### ISSUE TITLE
Why does the training stop automatically after about 7 to 10 epochs?

### ISSUE THREAD

**Issue by glorioustory** [strvcom/strv-ml-mask2face#6]
Hi, thanks for the work!
I train the model use tensorflow2.2.0-gpu, datasets contain about 39000 images, input dataset with mask and output dataset without mask, After 7-10 normal epochs, the program exits directly without prompt, The final print is after the test.

33/33 [==============================] - 11s 321ms/step - loss: 0.0150 - acc: 0.1446 - recall: 0.9332 - precision: 1.0000
libpng warning: iCCP: known incorrect sRGB profile
libpng warning: iCCP: known incorrect sRGB profile
- TEST -> LOSS:     0.0150, ACC:     0.1446, RECALL:     0.9332, PRECISION:     1.0000

Other exits are also in this place. Why is this?
